packages/flutter/test/painting/gradient_test.dart

```diff
@@ -91,11 +92,13 @@ void main() {
       end: Alignment(-1.0, 0.0),
       colors: <Color>[
         Color(0x3B3B3B3B),
+        Color(0x55555555),
```

Comment:
Instead of editing an existing test case please add new tests specific to this change and keep the old test cases as-is to ensure that those cases keep working as well with your change.

Reply:
Maybe I'm misunderstanding you, but I changed the behaviour so the test is still valid but the expected output has changed.

Reply:
Oh, sorry. I wasn't looking at the full context here.